### PR TITLE
minor changes in documentation

### PR DESCRIPTION
**Pull request by oindrillac**
Added some urls to add more context to the README documentation which is also published on the website.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,17 +1,18 @@
...
-Kustomize 3.8.1+
-SOPS 3.4.0+
-KSOPS 2.1.2+
+* Kustomize 3.8.1+
+* SOPS 3.4.0+
+* KSOPS 2.1.2+
+Find more informaton on how to install [here](docs/manage_your_app_secrets.md).
 
-Ensure you have the key to decrypt secrets. Reach out to members of the Data Hub team for access.
+Ensure you have the key to decrypt secrets. Reach out to members of the Data Hub team(data-hub-users@redhat.com) for access.
```
Comment (anishasthana) [inline README.md]: I get the feeling this email is wrong.

**Review by anishasthana — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (oindrillac) [reply README.md]: do you know the right one? is it perhaps, data-hub@redhat.com?

I think I'm subscribed to this one where the users ask questions
image

Comment (anishasthana) [reply README.md]: Well, we generally  ask folks to email `data-hub@redhat.com`. That being said, these docs are for general operate-first right? data-hub team is the wrong one to refer to here I think. @HumairAK am i mistaken?

Comment (oindrillac) [reply README.md]: So should we completely remove that then? Probably ask them to open an issue in this repo?

Comment (anishasthana) [reply README.md]: I think changing it to say "open an issue in this repo" is the right call

Comment (oindrillac) [reply README.md]: Done

File: README.md [hunk 2]
```diff
@@ -1,17 +1,19 @@
...
-SOPS 3.4.0+
-KSOPS 2.1.2+
+* Kustomize 3.8.1+
+* SOPS 3.4.0+
+* KSOPS 2.1.2+  
 
-Ensure you have the key to decrypt secrets. Reach out to members of the Data Hub team for access.
+Find more information on how to install the pre-requisites [here](docs/manage_your_app_secrets.md).
+
+Ensure you have the key to decrypt secrets. Open an issue in this [repository](https://github.com/operate-first/continuous-deployment/issues) requesting for access.
```
Comment (anishasthana) [inline README.md]: `requesting for access.` -> `requesting access.`

**Review by anishasthana — CHANGES_REQUESTED** [review]

**Comment by anishasthana** [comment]
Can you fix the whitespace errors as well?

File: README.md (hunk 2 above)
Comment (oindrillac) [reply README.md]: Fixed above and fixed whitespace errors

**Review by anishasthana — APPROVED** [review]
:ship:

**Pull request merged**